docs/site/Component.md

```diff
@@ -83,6 +83,37 @@ const app = new RestApplication();
 app.component(AuthenticationComponent);
 ```
 
+## Official components
+
+Here is a list of components officially created and maintained by the LoopBack
```

Comment:
I see the list below contains the components from the core (e.g. `@loopback/rest`) and extensions (e.g. `@loopback/apiconnect`). Perhaps it's better to separate the 2 categories?

Reply:
@bajtos what do you say?

Reply:
I don't mind either way.